lib/ecto/query.ex

```diff
@@ -475,11 +475,34 @@ defmodule Ecto.Query do
   @doc """
   Resets a previously set field on a query.
 
-  It can reset any query field except the query source (`from`).
+  It can reset many fields except the query source (`from`).
```

Comment:
Considering I discovered it couldn't update any query field such as inner_join, probably write "many" instead in case any else finds others that are missing.